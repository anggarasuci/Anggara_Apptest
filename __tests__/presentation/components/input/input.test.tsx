import React from 'react';
import {render} from '@testing-library/react-native';
import InputView from '../../../../src/presentation/components/input/input';

test('renders InputView correctly', () => {
  const onTextChange = jest.fn();

  const {toJSON} = render(
    <InputView
      label="Label"
      value=""
      placeholder="Placeholder"
      onTextChange={onTextChange}
    />,
  );

  expect(toJSON()).toMatchSnapshot();
});
