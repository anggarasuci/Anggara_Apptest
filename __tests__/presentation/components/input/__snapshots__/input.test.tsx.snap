// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders InputView correctly 1`] = `
<View
  style={
    {
      "marginBottom": 16,
    }
  }
>
  <Text
    style={
      [
        {
          "color": "#9E9E9E",
          "fontSize": 12,
          "fontWeight": "600",
        },
        null,
      ]
    }
  >
    Label
  </Text>
  <TextInput
    inputMode="none"
    onChangeText={[Function]}
    placeholder="Placeholder"
    style={
      [
        {
          "borderColor": "#e0e0e0",
          "borderRadius": 10,
          "borderWidth": 1.5,
          "height": 40,
          "marginBottom": 4,
          "paddingHorizontal": 10,
          "width": "100%",
        },
        null,
      ]
    }
    value=""
  />
</View>
`;
